plugins/providers/gcloudiam/errors.go

```diff
@@ -14,4 +14,6 @@ var (
 	ErrShouldHaveEmptyRoles          = errors.New("gcloud_iam provider should have empty roles")
```

Comment:
this can be removed since no longer being used everywhere
```suggestion
```

Reply:
Done